nsxt/resource_nsxt_policy_host_transport_node_collection.go

```diff
@@ -89,6 +89,12 @@ func resourceNsxtPolicyHostTransportNodeCollection() *schema.Resource {
 				Optional:    true,
 				Description: "Transport Node Profile Path",
 			},
+			"remove_nsx_on_destroy": {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Indicate whether NSX service should be removed from hypervisors during resource deletion",
```

Comment:
nit for description:

Upon deletion, uninstall NSX from Transport Node Collection member hosts.